.mvn/project-settings.xml

```diff
@@ -1,26 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I don't remember why this file was added previously, but if the tests in Travis works then it shouldn't be a problem to remove it 😄

Reply:
sorry,those were added unfortunately will remove those.